01-python/infrastructure/tree.py

```
@@ -1,3 +1,20 @@
-def draw_tree():
-    print("tree not found")
-    return
+'''
+    print a tree like 
+       o
+      ooo
+     ooooo
+       o
+'''
+def draw_tree(height = 5):
+    if (height == 0) :
+        print ("tree not found.")
```

Better to just return here

---

(instead of printing)

---

If the height <0 you should raise an error